src/Worker.Extensions.DurableTask/HTTP/DurableHttpResponse.cs

```diff
@@ -17,16 +17,10 @@ public class DurableHttpResponse
     /// Initializes a new instance of the <see cref="DurableHttpResponse"/> class.
     /// </summary>
     /// <param name="statusCode">HTTP Status code returned from the HTTP call.</param>
-    /// <param name="headers">Headers returned from the HTTP call.</param>
-    /// <param name="content">Content returned from the HTTP call.</param>
     public DurableHttpResponse(
```

Comment:
nit: can inline param now:
```suggestion
    public DurableHttpResponse(HtttpStatusCode statusCode)
```